content/projects/gsoc/2023/project-ideas/buiilding-ios-app.adoc

```diff
@@ -0,0 +1,59 @@
+---
+layout: gsocprojectidea
+title: "Building IOS Apps with Jenkins"
```

Comment:
Incorrect capitalization: [iOS](https://en.wikipedia.org/wiki/IOS) has a lowercase i.

Reply:
to be fixed with #5965

Reply:
Very nice!